src/model/Model.ts

```diff
@@ -372,7 +342,12 @@ export class Model {
    */
   protected $fillField(key: string, attr: Attribute, value: any): void {
     if (value !== undefined) {
-      this[key] = attr.make(value)
+      if (attr instanceof MorphTo) {
+        this[key] = attr.make(value, this[attr.$getType()])
```

Comment:
`MorphTo` needs to know the entity type of the child in order to correctly create a new instance